openassessment/xblock/validation.py

```diff
@@ -58,27 +63,41 @@ def _is_valid_assessment_sequence(assessments):
         bool
 
     """
-    valid_sequences = [
-        ['self-assessment'],
-        ['peer-assessment'],
-        ['peer-assessment', 'self-assessment'],
-        ['self-assessment', 'peer-assessment'],
-        ['student-training', 'peer-assessment'],
-        ['student-training', 'peer-assessment', 'self-assessment'],
-        ['student-training', 'self-assessment', 'peer-assessment'],
-        ['example-based-assessment'],
-        ['example-based-assessment', 'self-assessment'],
-        ['example-based-assessment', 'peer-assessment'],
-        ['example-based-assessment', 'peer-assessment', 'self-assessment'],
-        ['example-based-assessment', 'self-assessment', 'peer-assessment'],
-        ['example-based-assessment', 'student-training', 'peer-assessment'],
-        ['example-based-assessment', 'student-training', 'peer-assessment', 'self-assessment'],
-        ['example-based-assessment', 'student-training', 'self-assessment', 'peer-assessment'],
-    ]
-
     sequence = [asmnt.get('name') for asmnt in assessments]
-    return sequence in valid_sequences
+    required = ['staff-assessment', 'peer-assessment', 'self-assessment']
```

Comment:
Was it necessary to change this code? It is my understanding that staff assessment will always be at the end.